pyproject.toml

```diff
@@ -75,7 +75,7 @@ dependencies = [
     "webauthn",
     "xhtml2pdf==0.2.8",
     "reportlab<4.0.0",
-    "zammad-py",
+    "zammad-py"
```

Comment:
Interesting, I assume this somhoe comes from adding then removing `django-cron`? 

Just checked locally, and black/ruff don't seem to care about the comma either way, whereas they are opinionated about them everywhere else in the project. Huh.

I can't believe how nitpicky this is, but since all *other* lists in `pyproject.toml` end with a comma,... 😅 😂